scripts/mint-batch.ts

```diff
@@ -18,15 +17,17 @@ async function main(numMints: string) {
   const result = await client.execute(
     config.account,
     config.minter,
-    { batchMint: { numMints } },
-    executeFee
+    { batch_mint: { num_mints: Number(numMints) } },
+    executeFee,
+    "batch mint",
+    coins("100000000", "ustars")
```

Comment:
magic number?

Reply:
This are the mint fees and price. Will break out into variables.